package-lock.json

```diff
@@ -2,6 +2,14 @@
   "requires": true,
   "lockfileVersion": 1,
```

Comment:
Same for this file, it's the root one, please remove from this PR

Reply:
Which file do you mean here?